tests/util/kubeUtils.go

```diff
@@ -128,26 +127,23 @@ func KubeDelete(namespace, yamlFileName string) error {
 // GetIngress get istio ingress ip
 func GetIngress(n string) (string, error) {
 	retry := Retrier{
-		BaseDelay: 5 * time.Second,
-		MaxDelay:  20 * time.Second,
-		Retries:   20,
+		BaseDelay: 1 * time.Second,
+		MaxDelay:  1 * time.Second,
+		Retries:   300, // ~5 minutes
 	}
 	ri := regexp.MustCompile(`^[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$`)
 	//rp := regexp.MustCompile(`^[0-9]{1,5}$`) # Uncomment for minikube
 	var ingress string
 	retryFn := func(i int) error {
-		ip, err := Shell("kubectl get svc istio-ingress -n %s -o jsonpath='{.status.loadBalancer.ingress[*].ip}'", n)
+		ip, err := ShellSilent("kubectl get svc istio-ingress -n %s -o jsonpath='{.status.loadBalancer.ingress[*].ip}'", n)
```

Comment:
I can't tell from all the changes, let's look at the build output I guess, but it is useful to see the state of the system evolve as it retries
(but then I guess with 1s retry that may be a lot of output when it will fail after 300)

Reply:
https://gist.github.com/ayj/31b985d7e234f902c1ffaacb2b4d1f03#file-sample-logs4-txt-L92-L110